src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedAutoPropAccessorSymbol.cs

```diff
@@ -0,0 +1,204 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Microsoft.Cci;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class SynthesizedAutoPropAccessorSymbol : SynthesizedInstanceMethodSymbol
+    {
+        public enum AccessorKind : byte
+        {
+            Get,
+            Set,
+            Init
+        }
+
+        private readonly SourceOrRecordPropertySymbol _property;
+        private readonly AccessorKind _accessorKind;
+        private readonly StrongBox<TypeWithAnnotations>? _voidType;
+
+        public override string Name { get; }
+
+        public SynthesizedAutoPropAccessorSymbol(
+            SourceOrRecordPropertySymbol property,
+            string paramName,
+            AccessorKind accessorKind,
+            DiagnosticBag diagnostics)
+        {
+            _property = property;
+            _accessorKind = accessorKind;
+            Name = SourcePropertyAccessorSymbol.GetAccessorName(
+                paramName,
+                getNotSet: accessorKind == AccessorKind.Get,
+                isWinMdOutput: false /* PROTOTYPE */);
```

Comment:
>PROTOTYPE [](start = 40, length = 9)

Assuming we still want to try and get data properties into 16.7p3, we should replace this and any other PROTOTYPE markers with a link.